chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -128,6 +125,13 @@
 <!ENTITY zotero.toolbar.openURL.label					"Locate">
 <!ENTITY zotero.toolbar.openURL.tooltip				"Find through your local library">
 
+<!ENTITY zotero.toolbar.feeds.new					"New Feed…">
+<!ENTITY zotero.toolbar.feeds.new.fromURL			"From URL…">
+<!ENTITY zotero.toolbar.feeds.new.fromPage			"From Page">
+<!ENTITY zotero.toolbar.feeds.new.fromPage.menu		"From Page…">
```

Comment:
`zotero.toolbar.feeds.new.fromPage` is unused, I think, so can just change `fromPage.menu` to `fromPage`.